src/sass/star-rating-select/star-rating-select.scss

```diff
@@ -6,7 +6,7 @@ $color-star-rating-icon-unfilled-foreground: transparent;
 $color-star-rating-icon-filled-foreground: var(--color-foreground-primary);
 
 /* we have to use core color here since dark mode and light mode have the same stroke */
-$color-star-rating-icon-unfilled-stroke: var(--color-neutral-5);
+$color-star-rating-icon-unfilled-stroke: var(--color-neutral-600);
```

Comment:
Should we switch these to semantic tokens possibly?

Reply:
There's a comment right above that regarding the decision to use the core token instead of the semantic token.